extensions/resteasy-reactive/rest-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/ExceptionInReaderTest.java

```diff
@@ -24,7 +24,7 @@ public JavaArchive get() {
 
     @Test
     public void test() {
-        RestAssured.with().contentType("application/json").body("{\"name\": \"brie\"}").put("/fromage")
```

Comment:
How was it supposed to trigger an error before?

Reply:
It would trigger an error because no matching constructor existed - Jackson has now become smarter  and doesn't need `@JsonCreator` so I needed to find another way to trigger an mapping exception.

Reply:
Makes perfect sense, thanks!

Reply:
🙏🏽